src/transcription/schema.ts

```diff
@@ -0,0 +1,162 @@
+import { z } from 'zod'
+import { Status, TranslationLanguage } from './types'
+
+export const GetTranscriptionRequestSchema = z
```

Comment:
why do we keep it capitalized?

Reply:
Why not? All schema namings are capitalized

Reply:
It aligns well with TypeScript conventions for types and interfaces. For example, naming your schemas as `GetTranscriptionRequestSchema` clearly indicates that these are schema definitions rather than ordinary variables.

Reply:
Types and Interfaces are types and schema is a variable - they have different purposes, and shouldn't be aligned in syntax, from my perspective.

on the other side, as I mentioned before - it's a question of a project code styles, so the comment is just to bring attention.

Reply:
So let's keep schema names capitalized as projects code styles